### PR TITLE
Add python-billiard to pulp-scl

### PR DESCRIPTION
**Pull request by pcreech**
Add python-billiard to pulp's scl

re #[2338](https://pulp.plan.io/issues/2338)

### CODE REVIEW

File: rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Release:	1%{?dist}
+Summary:	Multiprocessing Pool Extensions
+
+License:	BSD
+URL:		http://pypi.python.org/pypi/billiard
+Source0:	http://pypi.python.org/packages/source/b/%{srcname}/%{srcname}-%{version}.tar.gz
+
+BuildRequires:	%{?scl_prefix_python}python-devel
+BuildRequires:	%{?scl_prefix_python}python-setuptools
+BuildRequires:	%{?scl_prefix_python}python-nose
```
Comment (werwty) [inline rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec]: Do we need python-nose in order to build? This seems to be more of a devel dependency...

File: rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+%{?scl:BuildRequires:	%{scl}-build %{scl}-runtime}
+
+%{?scl:Requires:	%{scl}-runtime}
+
+BuildRoot:      %{_tmppath}/%{pkg_name}-%{version}-%{release}-root-%(%{__id_u} -n)
```
Comment (werwty) [inline rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec]: I'm not sure if we need this. I'm not seeing this being referenced to in the build log. And according to https://fedoraproject.org/wiki/How_to_create_an_RPM_package

> BuildRoot: This is where files will be "installed" during the %install process (after the %build process). This is now redundant in Fedora and is only needed for EPEL5. By default, the build root is placed in "%{_topdir}/BUILDROOT/".

**Review by werwty — CHANGES_REQUESTED** [review]

File: rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec (hunk 2 above)
Comment (pcreech) [reply rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec]: Correct, we don't.  I removed the overridden build root lines from other pakcages, it appears I left it in here

File: rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec (hunk 1 above)
Comment (pcreech) [reply rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec]: Should be able to remove, as we don't do anything with %check, for now at least.

**Review by werwty — APPROVED** [review]

**Pull request merged**